<div class="faqPage">
  <div class="title-row">
    <div class="row">
      <div class="col-xs-12">
        <div class="name">
          <h3>Frequently Asked Questions</h3>
        </div>
      </div>
    </div>
  </div>
  <div class="row">
    <div class="col-xs-12 col-xs-offset-0 col-md-10 col-md-offset-1 col-lg-8 col-lg-offset-2">
      <p>
        <strong><i>What is CIViC?</i></strong><br/>

        CIViC is an open access, open source, community-driven web resource for Clinical Interpretation of Variants in Cancer. Our goal is to enable precision medicine by providing an educational forum for dissemination of knowledge and active discussion of the clinical significance of cancer genome alterations.</p>

      <p><strong><i>What is precision medicine?</i></strong><br/>

        Precision medicine refers to the use of prevention and treatment strategies that are tailored to the unique features of each individual and their disease. In the context of cancer this might involve the identification of specific mutations known to predict response to a targeted therapy. You can read more about this topic in summaries from the <a href="http://www.whitehouse.gov/the-press-office/2015/01/30/fact-sheet-president-obama-s-precision-medicine-initiative">White House</a> and the <a href="http://www.nejm.org/doi/full/10.1056/NEJMp1500523#t=article">New England Journal of Medicine</a>.</p>

      <p><strong><i>Why crowdsourcing?</i></strong><br/>

        The rapid expansion of the biomedical literature presents an intractable challenge for investigators and clinicians to interpret and synthesize, in real time, the complete array of knowledge available for each cancer-associated genomic variant into a clinically-relevant interpretation. Efforts to curate the clinical importance of variants have been undertaken by a diverse, but fragmented, community of individuals. Crowdsourcing leverages the resources of individuals to produce a centralized platform for this diverse community to discuss and share findings in their specialities. CIViC also serves as a platform for these contributors keep up to date on recent advances.</p>

      <p><strong><i>Who should be using CIViC?</i></strong><br/>

        CIViC is intended to bring together a diverse group of individuals interested in the clinical application and advancement of cancer genomics including health professionals (oncologists, pathologists and other clinicians), researchers (cancer genome and data scientists), and patient advocates.</b>

      <p><strong><i>What is a <b>Variant</b>?</i></strong><br/>

        A variant in CIViC is any genomic alteration associated with cancer. This includes single nucleotide variants (SNVs), insertion/deletion events (indels), copy number alterations (CNV's), structural variants (SVs), transcript fusions, and other genomic/molecular events with cancer associations, including germline variants.</p>

      <p><strong><i>What is an <b>Evidence Statement</b>?</i></strong><br/>

        An evidence statement is the result of an experiment, trial or study from a published literature source. It captures a variant's impact on clinical action, which can be predictive of therapy, correlated with prognostic outcome, or inform disease diagnosis. A single citation can be the source of multiple evidence statments.</p>

      <p><strong><i>How is this information organized?</i></strong><br/>

        An in-depth description of the data model behind CIViC can be found <a ui-sref="help.evidence"> here</a>.</p>

      <p><strong><i>What information is currently in CIViC?</i></strong><br/>

        CIViC currently houses over 450 evidence statements about 125 variants in 50 genes across 40 cancer types, and these numbers are growing rapidly with your help!</p>

      <p><strong><i>How do I contribute information?</i></strong><br/>

        The <a ui-sref="collaborate">Collaborate</a> tab details the curation process, with links for adding and editing information.</p>

      <p><strong><i>My favorite Gene/Variant is not in CIViC?!?</i></strong><br/>

        The manual curation involved in adding evidence statements means we have not been able to populate a published evidence statement from every variant or gene of interest. This is precisely why we need community-driven efforts to grow this database. In order for a gene or variant to appear in CIViC, it must have an evidence statement associated with it. To add evidence statements and begin the discussion about your gene/variant of interest, go <a ui-sref="collaborate">here</a>!</p>

      <p><strong><i>What kinds of variants belong in CIViC? What does not?</i></strong><br>

        In order to be accepted by CIViC, a variant must be related to cancer and have some *clinical* relevance. Relevance to cancer biology alone is insufficient unless there is also some documented relationship between the variant and diagnosis, prognosis, or response to a specific treatment. Variants related to diseases other than cancer should not be entered. Most variants in CIViC are somatic but germline variants are also welcome (as long as they have diagnostic, prognostic or predictive value). The quality of evidence suggesting clinical relevance of a variant may vary considerably. Before contributing to CIViC please review the <a ui-sref="help.evidence"> data model</a>.</p>

      <p><strong><i>May I enter unpublished results from n-of-1 observations?</i></strong><br>

        At this time, a publication is considered a minimum requirement for all evidence statements in CIViC. N-of-1 results from early stage clinical trials or patients treated under compassionate use doctrines are allowed, but only if a case report has been published in a peer reviewed journal. We are considering options for centers that wish to use a local instance of CIViC to capture individual patient observations.

      <p><strong><i>Why must my changes be approved by a moderator before I can see them?</i></strong><br/>

        In an effort to ensure quality (and prevent automated spam), we require that edits be submitted to the review queue before they are shown on the live server. Moderators are used for this review step to protect the CIViC resource and will approve your changes as soon as possible.</p>

      <p><strong><i>What is a Disease Ontology ID (DOID)?</i></strong><br/>

        In order to provide a structured representation of the diseases associated with evidence statements, we ask that you provide a Disease Ontology ID from <a href="http://disease-ontology.org/"> disease-ontology.org</a>. This allows for consistent representation and  minimized ambiguity when referring to diseases. Such ontologies also support more flexible data queries that allow disease groupings ranging from generic terms to highly specific subtypes.</p>

      <p><strong><i>What is a PubChem ID (PCID)?</i></strong><br/>

        Providing a PubChem ID with your evidence statement allows us to minimize ambiguity within drug naming. As a default, drug compound names are used instead of branded trade names in an effort to be as consistent across experiment types as possible. PubChem ID's can be found by searching the compound <a href="https://pubchem.ncbi.nlm.nih.gov/"> here</a>. However, when a PCID is not available, "N/A" is accepted.</p>

      <p><strong><i>Where does this information come from? Can I submit my abstract?</i></strong><br/>

        The information in CIViC is derived from peer-reviewed, published literature. Every evidence item currently requires a citation from a PubMed ID. This means that conference abstracts are not supported until they are published and peer-reviewed with a PubMed ID.

      <p><strong><i>What does the "Beta" label on CIViC mean?</i></strong><br/>

        This is in reference to the current software release state of CIViC. The "Beta" state indicates we are currently in the software testing phase with additional features and content actively being added.</p>

      <p><strong><i>What if a drug is shown to have a negative effect on patients with a variant? Or the study was inconclusive?</i></strong><br/>

        The data model, specifically the Evidence Direction field, is used to indicate whether the study supports or refutes (including inconclusive determinations) any interaction between the variant and a clinical action or result. The Clinical Significance field indicates the type of effect the variant is determined to have on clinical results within the evidence statement as having a positive, negative or neutral/no impact. These descriptions provide human readible interpretations of evidence statements that either support or refute sensitivity or resistance predictions to therapeutics (or other clinical outcomes). For more detailed definitions and specific examples, please review the data model <a ui-sref="help.evidence"> here</a>.</p>

      <p><strong><i>My evidence statement disagrees with evidence from another source, should I still add it?</i></strong><br/>

        Absolutely. CIViC provides a forum for discussion of disagreements in the field or literature. Simply log in, go to the "Variant Talk" or "Evidence Talk" pages, and discuss this disagreement with the community.</p>

      <p><strong><i>I don't want to add evidence statements, can I still contribute?</i></strong><br/>

        Yes. Evaluation of the literature is a collaborative effort. If you don't want to add new evidence you can rate, edit or discuss existing evidence. You can also help to make sure the Variant Summary is an effective, consise, and accurate summary of the current set of evidence statements for each variant.</p>

      <p><strong><i>What is MyGene.info?</i></strong><br/>

        <a href=http://mygene.info/> MyGene.info</a> is a web service that allows simple query and retrieval of gene annotation data. We use it in CIViC to automatically import gene details from Entrez Gene (<a href=http://www.ncbi.nlm.nih.gov/gene>http://www.ncbi.nlm.nih.gov/gene</a>) such as gene name, synonyms, protein domains and pathways. MyGene.info was created and is maintained by the <a href=http://sulab.org/>Su Lab</a> (at The Scripps Research Institute).

      <p><strong><i>Is there an API I can access?</i></strong><br/>

        Yes! Please visit the <a href="https://civic.genome.wustl.edu/#/api-documentation">API documentation</a> for more details.

      <p><strong><i>What is the difference between "Supports a Negative association" and "Does not support a Positive association"?</i></strong><br/>

        This can be confusing. We have reserved "Does not support - Positive" for statements that contradict previous statements that are supporting positive associations. For example, they would read "Contrary to the previous study which found this mutation sensitive to drug X, this study reported no effect.". The information that this study "Does not support" the prior study's conclusion is what we are trying to capture with these classifications.

      <p><strong><i>How is CIViC licensed?</i></strong><br/>

        The content of CIViC, hosted by <a href="http://genome.wustl.edu/">The McDonnell Genome Institute at Washington University School of Medicine</a> is licensed under a <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a> and the source code for the CIViC <a href="https://github.com/genome/civic-server">server</a> and <a href="https://github.com/genome/civic-client">client</a> is licensed under the <a href="http://opensource.org/licenses/MIT">MIT License</a>.

      <p><strong><i>Where do I begin to curate evidence statements? Are certain evidence types or genes a priority?</i></strong></br>

        The evidence statements that make up CIViC are generated from peer-reviewed, published literature. Our top priority is high quality (4+ star rating) Level A and B evidence that associates specific variants from patient data directly with clinical outcomes. To help direct users towards genes known to be associated with clinical outcomes, we have compiled a list of high priority genes which you can download <a href="https://github.com/genome/civic-server/tree/master/public/downloads/RankedCivicGeneCandidates.tsv">here</a>.
 
    </div>
  </div>
</div>
