<div class="help-sub-tab">
  <p>
  CIViC variants are usually genomic alterations, including single nucleotide variants (SNVs), insertion/deletion events (indels), copy number alterations (CNV's such as amplification or deletion), structural variants (SVs such as translocations and inversions), and other events that differ from the "normal" genome. In some cases a CIViC variant may represent events of the transcriptome or proteome. For example, 'expression' or 'over-expression' is a valid variant. Regardless of the variant, it must have an Predictive, Prognostic, Predisposing or Diagnostic clinical relevance to be entered in CIViC. New curators should generally avoid proposing new variants that are unlike any already in CIViC.

  <h4>Variant Attributes</h4>

  <table class="table table-striped table-condensed table-bordered">
    <tr>
      <th class="attribute">
        Attribute
      </th>
      <th class="description">
        Description
      </th>
      <th class="source">
        Source
      </th>
    </tr>
    <tr>
      <td class="attribute">
        Name
      </td>
      <td class="description">
        Description of the type of variant (e.g., 'V600E', 'BCR-ABL fusion', 'Loss-of-function', 'Exon 12 mutations'). Should be as specific and short as possible (i.e., specific amino acid changes).
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Summary
      </td>
      <td class="Summary">
        User-defined summary of the clinical relevance of this Variant.
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Alias
      </td>
      <td class="description">
        Alternative names for this Variant. May be more verbose (e.g., 'Val600Glu') versions, dbSNP IDs or alternative nomenclature used in the literature.
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr>
      <td class="attribute">
        HGVS Expression
      </td>
      <td class="Summary">
        User-defined HGVS strings following <a href='http://varnomen.hgvs.org/'>HGVS nomenclature</a> that represent this Variant at the DNA, RNA or protein level.
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr>
      <td class="attribute">
        ClinVar ID
      </td>
      <td class="description">
        User-defined ClinVar ID referencing this Variant which will be linked directly to ClinVar.
        A value of "None Specified" indicates that the variant has not been evaluated for a ClinVar ID.
        A value of "None Found" indicates that an attempt was made to find a matching ClinVar Entry, but none exists.
        A value of "N/A" indicates that a ClinVar record is not applicable to the variant (e.g. "Overexpression" variants).
      </td>
      <td class="source">
        CIViC (ClinVar)
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Sources
      </td>
      <td class="Summary">
        A list of PubMed IDs referring to evidence supporting statements made in the Gene's description. Source descriptions (e.g. <em>'Weisberg et al., 2007, Nat. Rev. Cancer'</em>) are pulled from the PubMed database at the time of submission, and are not editable.
      </td>
      <td class="source">
        CIViC (PubMed)
      </td>
    </tr>
    <tr>
      <td class="attribute">
	      CIViC Variant Evidence Score
      </td>
      <td class="description">
        The CIViC Variant Evidence Score assesses the quality and quantity of evidence submitted for each variant. The CIViC Variant Evidence Score is calculated by adding all Evidence Item Scores for each variant. The Evidence Item Score is calculated by multiplying the evidence level (A=10 points, B=5 points, C=3 points, D=1 point, E=0.25 points) by the evidence rating (Each Star = 1 point).
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Variant Type(s)
      </td>
      <td class="description">
        One or more terms from the Sequence Ontology that describes the type of variant. Should be the most descriptive term applicable on a given branch (e.g., 'Conservative Missense Variant', 'Stop Gained', 'Transcript Fusion').
      </td>
      <td class="source">
        CIViC (Sequence Ontology)
      </td>
    </tr>
    <tr class="subheader">
      <td colspan="4">
        <h4>Primary Coordinates</h4>
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Reference Build
      </td>
      <td class="description">
        NCBI or GRC human reference assembly version.
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Chromosome
      </td>
      <td class="description">
        Name of the chromosome in which the variant occurs.
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Start, Stop
      </td>
      <td class="description">
        Start and stop positions of the variant (1-based genome coordinate). Start must be less than or equal to stop.
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Reference & <br/> Variant Bases
      </td>
      <td class="description">
        The nucleotide base of the reference and variant allele (e.g. 'C', 'A').
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>

    <tr>
      <td class="attribute">
        Representative Transcript
      </td>
      <td class="description">
        Ensembl transcript ID and version number for a known transcript of the gene that contains the variant (e.g. 'ENST00000263967.3').
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr class="subheader">
      <td colspan="4">
        <h4>Secondary Coordinates</h4>
      </td>
    </tr>
    <tr>
      <td class="attribute">
        Same as Primary
      </td>
      <td class="description">
        For fusion variants, the secondary coordinates are used to specify the 3' partner of the fusion gene.
      </td>
      <td class="source">
        CIViC
      </td>
    </tr>
    <tr class="subheader">
      <td colspan="4">
        <h4>MyVariant.Info</h4>
      </td>
    </tr>
    <tr>
      <td class="attribute">
        MyVariant Info
      </td>
      <td class="description">
        Data retrieved from MyVariant.Info using the curated Primary Coordinates (chromosome, start, stop, reference base, variant base) described above as the query. Includes external IDs and links whenever possible with additional data displayed by clicking the "Details" button.
      </td>
      <td class="source">
        MyVariant.Info
      </td>
    </tr>
  </table>
</div>
