<div class="help-sub-tab">
  <p>
  Variant types are used to classify variants within the structured ontology terms provided by <a href='http://www.sequenceontology.org/index.html'>The Sequence Ontology Project</a>. Multiple terms for a variant are allowed to facilitate functional and structural effects of the variant. Ideally, the most specific child terms of a branch are used. These variant types can be used to search for variants within the CIViC database that meet a users' criteria via the advanced search page or through the API.</p>
  <p>
  General Rules:<br>
  1) Choose the most specific terms.<br>
  2) Do not use terms that are ancestors/decendants of each other.<br>
  3) Wherever possible use the 'sequence_variant' tree of the sequence ontology.</p>

  <h4>Variant Type Guidelines</h4>

  <table class="table table-striped table-condensed table-bordered">
    <tr>
      <th class="term">
        Sequence Ontology Term
      </th>
      <th class="definition">
        Sequence Ontology Definition
      </th>
      <th class="comments">
        Comments
      </th>
      <th class="examples">
        Examples
      </th>
    </tr>
    <tr>
      <td class="term">
        missense_variant
      </td>
      <td class="definition">
        A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/30/summary/variants/79/summary'>G12D</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        protein_altering_variant
      </td>
      <td class="definition">
        A sequence_variant which is predicted to change the protein encoded in the coding sequence.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/30/summary/variants/76/summary#variant'>G12</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/20/summary/variants/413/summary'>KINASE DOMAIN MUTATION</a></p>
      </td>
    </tr>
    <tr>
      <td class="term">
        frameshift_truncation
      </td>
      <td class="definition">
        A frameshift variant that causes the translational reading frame to be shortened relative to the reference feature.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/69/summary/variants/243/summary'>V2288fs*1</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        inframe_deletion
      </td>
      <td class="definition">
        An inframe non synonymous variant that deletes bases from the coding sequence.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/38/summary/variants/101/summary'>DEL I843</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/29/summary/variants/202/summary'>V560DEL</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/20/summary/variants/37/summary'>DEL 755-759</a></p>
      </td>
    </tr>
    <tr>
      <td class="term">
        inframe_insertion
      </td>
      <td class="definition">
        An inframe non synonymous variant that inserts bases into in the coding sequence.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/20/summary/variants/41/summary'>P780INS</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/20/summary/variants/414/summary'>M774INSAYVM</a></p>
      <p><a href='https://civic.genome.wustl.edu/#/events/genes/24/summary/variants/55/summary'>ITD</a></p>
      </td>
    </tr>
    <tr>
      <td class="term">
        <p>[ gene_variant<br> OR<br> transcript_variant ]</p>
        <p>AND</p>
        <p>[ loss_of_function_variant<br> OR<br> gain_of_function_variant ]</p>
      </td>
      <td class="definition">
        <p>[ gene_variant: A sequence variant where the structure of the gene is changed.<br>
        OR<br> 
        transcript_variant: A sequence variant that changes the structure of the transcript ]</p>

        <p>AND</p> 

        <p>[ loss_of_function_variant: A sequence variant whereby the gene product has diminished or abolished function.<br>
        OR <br>
        gain_of_function_variant: A sequence variant whereby new or enhanced function is conferred on the gene product. ]</p>
      </td>
      <td class="comments">
        Depends on situation.
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/5/summary/variants/399/summary'>MUTATION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        exon_variant
      </td>
      <td class="definition">
        A sequence variant that changes exon sequence.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/37/summary/variants/106/summary'>EXON 10 MUTATION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        <p>transcript_fusion</p> 
        <p>OR RARELY...</p>
        gene_fusion
      </td>
      <td class="definition">
        <p>transcript_fusion: A feature fusion where the deletion brings together transcript regions.</p> 
        <p>OR</p>
        <p>gene_fusion: A sequence variant whereby a two genes have become joined.</p>
      </td>
      <td class="comments">
      <p>Depends on situation.</p>
      <p>Note that transcript_fusion mentions "deletion" specifically as the genomic alteration, and both gene_ and transcript_fusion are children of <a href='http://www.sequenceontology.org/browser/current_svn/term/SO:0001882'>feature_fusion</a>, which also mentions "deletion" specifically. However, it is assumed that deletion is just one possible mechanism (along with translocation, inversion, etc) for bringing two gene or transcribed regions together. The decision of which term to use therefore rests on the level of specificity. If the genomic event is thought to result in a fusion transcript then "transcript_fusion" is the preferred term.</p>
      </td>
      <td class="examples">
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/1/summary/variants/5/summary'>EML4-ALK</a></p>
        <a href='https://civic.genome.wustl.edu/#/events/genes/1/summary/variants/499/summary'>ALK FUSIONS</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        <p>transcript_fusion</p>
        <p>AND</p>
        <p>missense_variant</p>
      </td>
      <td class="definition">
        <p>transcript_fusion: A feature fusion where the deletion brings together transcript regions.</p> 
        <p>AND<p> 
        <p>missense_variant: A sequence variant that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.</p>
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/1/summary/variants/308/summary#variant'>EML4-ALK G1269A</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        <p>transcript_translocation</p>
        <p>OR</p>
        <p>feature_translocation</p>
        <p>OR</p>
        transcript_fusion
      </td>
      <td class="definition">
        <p>transcript_translocation: A feature translocation where the region contains a transcript.</p>
        <p>OR</p>
        <p>feature_translocation: A sequence variant, caused by an alteration of the genomic sequence, where the structural change, a translocation, is greater than the extent of the underlying genomic features.</p>
        <p>OR</p>
        <p>transcript_fusion: A feature fusion where the deletion brings together transcript regions.</p> 
      </td>
      <td class="comments">
        Depends on situation.
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/4941/summary/variants/269/summary'>REARRANGEMENT</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        wild_type
      </td>
      <td class="definition">
        An attribute describing sequence with the genotype found in nature and/or standard laboratory stock.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/5/summary/variants/426/summary'>WILD TYPE</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        loss_of_heterozygosity
      </td>
      <td class="definition">
        A functional variant whereby the sequence alteration causes a loss of function of one allele of a gene.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/4645/summary/variants/302/summary'>LOH</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        transcript_amplification
      </td>
      <td class="definition">
        A feature amplification of a region containing a transcript.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/8/summary/variants/18/summary'>AMPLIFICATION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        transcript_ablation
      </td>
      <td class="definition">
        A feature ablation whereby the deleted region includes a transcript feature.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/73/summary/variants/200/summary'>DELETION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        copy_number_change
      </td>
      <td class="definition">
        A sequence variant where copies of a feature (CNV) are either increased or decreased.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/19/summary/variants/191/summary'>COPY NUMBER VARIATION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        loss_of_function_variant
      </td>
      <td class="definition">
        A sequence variant whereby the gene product has diminished or abolished function.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/46/summary/variants/125/summary'>LOSS-OF-FUNCTION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        loss_of_function_variant...?<br>
        transcript_ablation...?
      </td>
      <td class="definition">
        <p>loss_of_fuction_variant: A sequence variant whereby the gene product has diminished or abolished function.</p>
        <p>transcript_ablation: A feature ablation whereby the deleted region includes a transcript feature.</br> 
      </td>
      <td class="comments">
        Depends on situation.
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/916/summary/variants/555/summary#variant'>LOSS</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        exon_loss_variant
      </td>
      <td class="definition">
        A sequence variant whereby an exon is lost from the transcript.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/52/summary/variants/324/summary'>EXON 14 SKIPPING MUTATION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        5_prime_UTR_variant
      </td>
      <td class="definition">
        A UTR variant of the 5' UTR.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/1741/summary/variants/255/summary'>5' UTR MUTATION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        3_prime_UTR_variant
      </td>
      <td class="definition">
        A UTR variant of the 3' UTR.
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <a href='https://civic.genome.wustl.edu/#/events/genes/29/summary/variants/256/summary'>3' UTR MUTATION</a>
      </td>
    </tr>
    <tr>
      <td class="term">
        N/A
      </td>
      <td class="definition">
      </td>
      <td class="comments">
      </td>
      <td class="examples">
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/8/summary/variants/19/summary'>EXPRESSION</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/9171/summary/variants/340/summary'>NUCLEAR EXPRESSION</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/1883/summary/variants/447/summary'>CYTOPLASMIC EXPRESSION</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/8/summary/variants/20/summary'>OVEREXPRESSION</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/69/summary/variants/179/summary'>UNDEREXPRESSION</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/3532/summary/variants/538/summary#variant'>METHYLATION</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/34/summary/variants/85/summary'>PROMOTER METHYLATION</a></p>
        <p><a href='https://civic.genome.wustl.edu/#/events/genes/14/summary/variants/27/summary'>PROMOTER HYPERMETHYLATION</a></p>
      </td>
    </tr>
 </table>
</div>
