<div class="help-sub-tab">
  <p>
  Genomic coordinates in CIViC are a single representation of a genomic alteration that can lead to a specific variant. Multiple genomic alterations may lead to the same variant (i.e., several changes at the basepair level can lead to the same amino acid change); however, CIViC currently provides only one possible example per variant.

  <h4>General Guidelines</h4>
  <ul>
    <li>Choose one representative variant per gene.</li>
    <li>Coordinates are 1-based.</li>
    <li>Currently we prefer variants from reference genome build 37.</li>
    <li>Representative transcripts are from Ensembl.</li>
    <ul>
        <li>Include Ensembl transcript ID and transcript version (e.g., 'ENST00000078429.1') and the Ensembl database version (e.g., 75). Refer to further documentation below for guidance on how to select a representative transcript.</li>
    </ul>
    <li>The start is always the numerically smaller coordinate.</li>
    <li>Complexities can be documented in the variant comments section.</li>
    <li>For fusions:</li>
    <ul>
	<li>The fusion variant is entered under only one gene. Choose the 'important' gene. Often, but not always, this is the 3' gene.</li>
	<li>The variant name is always 5' partner - 3' partner (e.g. BCR-ABL1, EML4-ALK, etc.) </li>
    </ul>
  </ul>

  <h4>Selecting a representative transcript</h4>
  <p>A representative transcript is a single transcript model used as an example or reference point. It is also used to define the genomic coordinates for the variant. We present a single representative transcript that contains the variant of interest that ideally can be exported using the API and converted bioinformatically to desired outputs.</p>
  <p>Features of a representative transcript:</p>
  <ul>
    <li>There is no one 'right' answer for representative transcript.</li>
    <li>It must:</li>
        <ul>
        <li>contain the variant (except in rare cases like promoter mutations).</li>
        </ul>
    <li>It should:</li>
        <ul>
        <li>be from Ensembl archived version 75 including the transcript version number (i.e., ENST00000078429.1 instead of ENST00000078429).</li>
        <li>be a transcript that is in our IGV reference transcript file. (see below)</li> 
        </ul>
    <li>It may:</li>
    <ul>
        <li>be the transcript with the longest ORF or most exons.</li>
        <li>be the transcript that contains the 'canonical exons’ that are used in many transcripts.</li>
        <li>be the variant that has the greatest outer coordinates.</li>
        <li>be the transcript that is widely used in literature.</li>
        <li>be a transcript that is compatible with interpretation/visualization in the literature (source paper for evidence statements).</li>
    </ul>
  </ul>

  <p>IGV reference transcript file
    <ul>
    <li>This file is a custom gene track file that can be loaded in IGV for review of coordinates.</li>
    <li>It is based on a specific version of Ensembl (v75) and contains both transcript IDs and transcript version numbers.</li>             
    <li>Ensembl defined ’<a href="http://www.ensembl.org/Help/Glossary?id=346">canonical transcripts</a>' are highlighted with asterisks (***ENST00000078429.1.v75***) as a suggestion when no specific transcript is otherwise indicated.</li>
    <li>The reference transcript file can be obtained from the <a href="https://github.com/genome/civic-server">civic-server git repository</a> here:<br><a href="https://raw.githubusercontent.com/genome/civic-server/master/public/downloads/Ensembl-v75_build37-hg19_UcscGenePred_CIViC-Genes.ensGene">Ensembl-v75_build37-hg19_UcscGenePred_CIViC-Genes.ensGene</a></li>
    </ul>

  <h4>Selecting variant coordinates based on variant type</h4>
  <p>Start and stop positions are dependent upon the reference genome and the specific isoform/transcript being used.</p>

  <table class="table table-condensed table-attributes">
    <tbody>
      <tr class="subheader">
        <td colspan="2">
          <h4>Single Nucleotide Variants (SNVs)</h4>
        </td>
      </tr>
      <tr>
        <td class="source">
          <img src="assets/images/ChoosingCoordinates_SNVs.png" class="img-fluid" width="45%" height="45%" alt="CIViC SNV coordinate diagram"/>
        </td>
        <td>
          <ul>
            <li>Start and stop coordinates are the same</li>
            <li>Secondary coordinates do not apply</li>
          </ul>
        </td>
      </tr>
      
      <tr class="subheader">
        <td colspan="2">
          <h4>Insertions</h4>
        </td>
      </tr>
      <tr>
        <td class="source">
          <img src="assets/images/ChoosingCoordinates_Insertions.png" class="img-fluid" width="45%" height="45%" alt="CIViC Insertion coordinate diagram"/>
        </td>
        <td>
          <ul>
            <li>Secondary coordinates do not apply</li>
          </ul>
        </td>
      </tr>

      <tr class="subheader">
        <td colspan="2">
          <h4>Deletions</h4>
        </td>
      </tr>
      <tr>
        <td class="source">          
          <img src="assets/images/ChoosingCoordinates_Deletions.png" class="img-fluid" width="45%" height="45%" alt="CIViC Ecosystem Diagram"/>
        </td>
        <td>
          <ul>
            <li>Secondary coordinates do not apply</li>
          </ul>
        </td>
      </tr>

      <tr class="subheader">
        <td colspan="2">
          <h4>Fusions</h4>
        </td>
      </tr>
      <tr>
        <td class="source">
          <img src="assets/images/ChoosingCoordinates_Fusions.png" class="img-fluid" width="65%" height="65%" alt="CIViC Ecosystem Diagram"/>
        </td>
        <td>
          <ul>
            <li>Secondary coordinates correspond to the 3' partner</li>
            <li>Stop1 and Start2 coordinates correspond to the edge of the nearest exon</li>
          </ul>
        </td>
      </tr>

      <tr class="subheader">
        <td colspan="2">
          <h4>Whole Gene alterations (altered expression, deletion, amplification) </h4>
        </td>
      </tr>
      <tr>
        <td class="source">
          <img src="assets/images/ChoosingCoordinates_Whole_Gene.png" class="img-fluid" width="65%" height="65%" alt="CIViC Ecosystem Diagram"/>
        </td>
        <td>
          <ul>
            <li>Secondary coordinates do not apply</li>
            <li>Variant coordinates include UTRs</li>
          </ul>
        </td>
      </tr>

      <tr class="subheader">
        <td colspan="2">
          <h4>Gene Segments ('hotspot' regions including exons or domains)</h4>
        </td>
      </tr>
      <tr>
        <td class="source">
          <img src="assets/images/ChoosingCoordinates_Gene_Segment.png" class="img-fluid" width="60%" height="60%" alt="CIViC Ecosystem Diagram"/>
        </td>
        <td>
          <ul>
            <li>Secondary coordinates do not apply</li>
          </ul>
        </td>
      </tr>

    </tbody>
  </table>
</div>
